src/CoWSwapEthFlow.sol

```diff
@@ -91,9 +91,11 @@ contract CoWSwapEthFlow is
             revert NotAllowedZeroSellAmount();
         }
 
+        address owner = msg.sender;
```

Comment:
uber-nano-please-ignore-nit: I think `msg.sender` is a `CALLER` opcode which is 2 gas vs using the need to create a stack variable as well as "retrieving" it (`DUP*` or `SWAP*`) which is 3 gas.

Peanuts, but I'm curious IMO doesn't significantly increase readability.